packages/pwa-kit-dev/src/configs/babel/babel-config.js

```diff
@@ -57,5 +57,22 @@ export default {
             presets: [require('@babel/preset-env'), require('@babel/preset-react')],
             plugins: [require('babel-plugin-dynamic-import-node-babel-7')]
         }
-    }
+    },
+    ignore: [
+        function (filepath) {
+            // Ignore node_modules, but include @salesforce/extension-* and pwa-kit-extension-sdk
+            if (/node_modules/.test(filepath)) {
+                // Only ignore non-extension @salesforce packages
+                if (
+                    /node_modules\/(?!@salesforce\/pwa-kit-extension-sdk|@salesforce\/extension-)/.test(
+                        filepath
```

Comment:
Unblocking comment,  let's look a bit further in the future when partners start to create their own extensions. Do they have to follow the pattern of @salesforce/extension-* or just extension-* or their brand name like this @company/extension?
If it is more than @salesforce/extension-* we may want to allow other  pattern here too to avoid bug later down the road.